### DISCUSSION TITLE
04/04/22 Meeting Agenda / Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [AngellXR/metaversedev#12] [Announcements]
# Agenda
- Welcome and Introductions
- Review [Readme](https://github.com/AngellXR/metaversedev)
- Review (quick) Github process
- Review [Milestones](https://github.com/AngellXR/metaversedev/milestones)
- Pick meeting time(s), next meeting 
- Ensure [everyone has access](https://github.com/AngellXR/metaversedev/settings)

Next month: 
- rapid working agreement
- bring a friend? 
- Happy Hour?
- Prototyping workshop / LDJ?

:tv: [Meeting Recording](https://drive.google.com/file/d/1OT3m6pD_yw3t0S94bhZmLkEexxsdPwRs/view?usp=drivesdk)
:speaking_head: [AngellXR Discord](https://discord.gg/BJHQP5Pwtf)

**Comment by mrmetaverse** [comment]
# HW 
- create a github account and send Jesse (@mrmetaverse) your username 
- review readme, anything missing? 
  - Review WHY, and WHO. 
  - ![image](https://user-images.githubusercontent.com/63426722/161637841-63b902ce-1fb9-475b-a09f-437f702ef0d7.png)
- how can we help others get involved as creators or consumers or this content?
  - raise awareness that this is available (or will be) for metaverse creators (or hopefuls)
  - raise awareness that this is available for the existing platforms/stakeholders looking to grow/hire/tell their story. 

## Future considerations: 
- Statement of outputs (WHAT on readme)
- Risk management (what's coming up, risk responses, risk owners, responses/actions)
- Content plan (i.e. SEMrush list of topics)

**Reply by cruzluna** [reply]
Great first meeting.

**Reply by mrmetaverse** [reply]
agreed! Thanks for being part of this thing

**Comment by msub2** [comment]
Any recording from this meeting?

**Reply by mrmetaverse** [reply]
Just added above 👆 thanks for the reminder!

**Reply by msub2** [reply]
Thanks! Would be interested to attend next month's meeting

**Reply by mrmetaverse** [reply]
We would love for you to be there! Bring a friend too if you'd like. We meet once a month.
